Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -569,14 +621,15 @@ private void saveControlPosition(int sharedPrefsId, int x, int y)
 	 * InputOverlayDrawableButton. Simply add it to the HashSet of overlay items and wait
 	 * for Android to call the onDraw method.
 	 * 
-	 * @param context  The current {@link Context}.
-	 * @param resId    The resource ID of the {@link Drawable} to get the {@link Bitmap} of.
-	 * @param buttonId Identifier for determining what type of button the initialized InputOverlayDrawableButton represents.
+	 * @param context        The current {@link Context}.
+	 * @param defaultResId   The resource ID of the {@link Drawable} to get the {@link Bitmap} of (Default State).
+	 * @param pressedResId   The resource ID of the {@link Drawable} to get the {@link Bitmap} of (Pressed State).
```

Comment:
There isn't an official coding style document for the Java code, but before your changes the spacing was only one away from the longest parameter, so I figured we should keep it that way. Also, you have it inconsistent right now (here the description is 3 spaces separated, but in other areas you have it as 2 or even 6 spaces apart). As an aside, OCD stands for Obsessive Compulsive Disorder and it means I want everything to be just right!